website/docs/streaming-lakehouse/integrate-data-lakes/paimon.md

```diff
@@ -99,7 +99,7 @@ This query may run slower than reading only from Paimon, but it returns the most
 
 ### Reading with other Engines
 
-Since the data tiered to Paimon from Fluss is stored as a standard Paimon table, you can use any engine that supports Paimon to read it. Below is an example using [StarRocks](https://paimon.apache.org/docs/master/engines/starrocks/):
+Since the data tiered to Paimon from Fluss is stored as a standard Paimon table, you can use any engine that supports Paimon to read it. Below is an example using [StarRocks](https://paimon.apache.org/docs/1.2/ecosystem/starrocks/):
```

Comment:
should we leave this master here? otherwise moving foward it might show outdated versions